Utils/gitlab_triggers/trigger_content_instance_testing_build.sh

```diff
@@ -1,22 +1,23 @@
-c# !/usr/bin/env bash
+#!/usr/bin/env bash
 # This script triggers an instance-testing job in gitlab-CI.
 
 # For this script to work you will need to use a trigger token (see here for more about that: https://code.pan.run/help/ci/triggers/README#trigger-token)  # disable-secrets-detection
 
 # This script takes the gitlab-ci trigger token as first parameter and the branch name as an optional second parameter (the default is the current branch).
 
 # Ways to run this script are:
-# 1. Utils/gitlab_triggers/trigger_content_nightly_build.sh <trigger-token> <branch-name>
-# 2. Utils/gitlab_triggers/trigger_content_nightly_build.sh <trigger-token>
+# 1. trigger_content_nightly_build.sh <trigger-token> <branch-name>
+# 2. trigger_content_nightly_build.sh <trigger-token>
```

Comment:
Isn't this the standard notation? (If you accept it, then you'll need to change the wording of the line above (# 8)
```suggestion
trigger_content_nightly_build.sh <trigger-token> [<branch-name>]
```